### PR TITLE
Bump version to v0.1.14

### PR DESCRIPTION
**Pull request by charliermarsh**

### CODE REVIEW

**Review by konstin — APPROVED** [review]

**Review by BurntSushi — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
+- Treat empty index URL strings as null instead of erroring ([#2137](https://github.com/astral-sh/uv/pull/2137))
+- Use space as delimiter for `UV_EXTRA_INDEX_URL` ([#2140](https://github.com/astral-sh/uv/pull/2140))
+- Report line and column numbers in `requirements.txt` parser errors ([#2100](https://github.com/astral-sh/uv/pull/2100))
+- Improve error messages when `uv` is offline ([#2110](https://github.com/astral-sh/uv/pull/2110))
+
+### Bug fixes
+
+- Future-proof the `pip` entrypoints special-case ([#1982](https://github.com/astral-sh/uv/pull/1982))
+- Allow empty extras in `pep508-rs` and add more corner case to tests ([#2128](https://github.com/astral-sh/uv/pull/2128))
+- Adjust base Python lookup logic for Windows ([#2121](https://github.com/astral-sh/uv/pull/2121))
```
Comment (AlexWaygood) [inline CHANGELOG.md:19]: Nit: It feels slightly unclear from this what impact the "adjustment" will have on users. Maybe this could be folded into the bullet in the "enhancements" section on adding support for Python installed from the Windows Store?

**Review by AlexWaygood — APPROVED** [review]
